aws-ssm-maintenancewindow/src/main/java/software/amazon/ssm/maintenancewindow/util/SimpleTypeValidator.java

```diff
@@ -0,0 +1,73 @@
+package software.amazon.ssm.maintenancewindow.util;
+
+import com.amazonaws.util.StringUtils;
+import software.amazon.awssdk.services.ssm.model.Tag;
+import software.amazon.awssdk.utils.CollectionUtils;
+
+import java.util.Optional;
+import java.util.List;
+import java.util.stream.Collectors;
+
+
+/**
+ * Validates data types that are not defined in Resource or Service models.
+ */
+public class SimpleTypeValidator {
```

Comment:
Well yes its is the re-use of SimpleTypeValidator, i can look into moving it.
I am not sure about the ClientBuilder though because it might have different retry value based on service or might not have any retry logic at all, hence something each service might want to customize there own off.

Reply:
Good point about ClientBuilder. Feel free to tab this off as a refactoring idea.